platform/src/pages/account/creation/step2.jsx

```diff
@@ -0,0 +1,142 @@
+                    required
+                  />
+                </div>
+              </div>
+              <div className='w-full'>
+                <div className='text-xs'>Last name*</div>
+                <div className='mt-2 w-full'>
+                  <input
+                    onChange={(e) => {
+                      setLastName(e.target.value);
+                    }}
+                    type='text'
+                    placeholder='e.g. Nagawa'
+                    className='input w-full rounded-none bg-form-input focus:outline-form-input focus:outline-none focus:outline-offset-0'
+                    required
+                  />
+                </div>
+              </div>
+            </div>
+          </div>
+          <div className='mt-6'>
+            <div className='w-full'>
+              <div className='text-xs'>Password*</div>
+              <div className='mt-2 w-full'>
+                <input
+                  onChange={(e) => {
+                    setPassword(e.target.value);
+                  }}
+                  pattern='^(?=.*?[A-Z])(?=.*?[a-z])(?=.*?[0-9])(?=.*?[#?!@$%^&]).{8,}$'
+                  title='Password should include an uppercase letter, lowercase letter, a number and special character(#?!@$^&)'
+                  minLength={'8'}
+                  maxLength={'20'}
+                  type='password'
+                  className='input w-full rounded-none bg-form-input focus:outline-form-input focus:outline-none focus:outline-offset-0'
+                  required
+                />
+              </div>
+            </div>
+          </div>
+          <div className='mt-6'>
+            <div className='flex flex-row items-center'>
+              <div>
+                <input
+                  type='checkbox'
+                  className='checkbox rounded border-2 border-check-box'
+                  required
+                />
+              </div>
+              <div className='ml-4'>
+                <p className='text-xs'>
+                  I agree to the{' '}
+                  <a href='#' className='underline'>
+                    Terms of Service
+                  </a>{' '}
+                  and{' '}
+                  <a href='#' className='underline'>
+                    Privacy Policy
+                  </a>
+                </p>
+              </div>
+            </div>
+          </div>
+          <div className='mt-14'>
+            <div className='flex flex-col-reverse md:flex-row items-center justify-start md:justify-between'>
+              <div className='w-full md:w-1/3 mt-6 lg:mt-0'>
+                <Link href='/account/creation'>
+                  <button className='w-full text-sm btn bg-white rounded-none outline-none border-2 border-check-box hover:bg-gray-100 hover:border-inherit focus:border-inherit'>
+                    <span style={{ color: '#353E52' }}>Back</span>
+                  </button>
```

Comment:
We do have a reusable button component that can be used for links/buttons. Kindly use that

Reply:
@Codebmk The Button component is not as reusuable because of how it's items are justified. Is that intended?
In the case that I want to justify my components with _justify-start_, does that require a new button component for all that different usecase?

Reply:
Okay. So far though the buttons we have are centered. If we need to create new button components to cater for those usecases, we can add them.